cli/network.go

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2018 Huawei Corporation.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/kata-containers/agent/protocols/grpc"
+	vc "github.com/kata-containers/runtime/virtcontainers"
+	"github.com/urfave/cli"
+)
+
+var networkCLICommand = cli.Command{
+	Name:  "network",
+	Usage: "manage interfaces and routes for container",
+	Subcommands: []cli.Command{
+		addIfaceCommand,
+		delIfaceCommand,
+		listIfacesCommand,
+		updateRoutesCommand,
+		listRoutesCommand,
+	},
+	Action: func(context *cli.Context) error {
+		return cli.ShowSubcommandHelp(context)
+	},
+}
+
+var addIfaceCommand = cli.Command{
+	Name:      "add-iface",
+	Usage:     "add an interface to a container",
+	ArgsUsage: `add-iface <container-id> file or - for stdin`,
+	Flags:     []cli.Flag{},
+	Action: func(context *cli.Context) error {
+		if context.Args().Present() == false {
+			return fmt.Errorf("missing container ID, should provide one")
+		}
+		containerID := context.Args().First()
```

Comment:
All these new `cli.Command`'s will require *two* calls to `setExternalLoggers()` to ensure logging is correct (see #528).

Reply:
Fixed